padrino/manifest.yml

```diff
@@ -0,0 +1,3 @@
+---
+applications:
+-  command: bundle exec padrino start -p $PORT
```

Comment:
"cf push requires an application name, which you provide either in a manifest or at the command line."

I created this manifest for a sinatra app which worked fine:
```
---
applications:
-  command: ruby server.rb
```